lib/App/Pinto/Command/add.pm

```diff
@@ -33,6 +33,8 @@ sub opt_spec {
         [ 'stack|s=s'                         => 'Put packages into this stack' ],
         [ 'use-default-message|M'             => 'Use the generated message' ],
         [ 'with-development-prerequisites|wd' => 'Also pull prereqs for development' ],
+        [ 'verify|Z'                          => 'Verify upstream files before processing'],
+        [ 'strict'                            => 'Make verification warnings fatal'],
```

Comment:
I'm not averse to long option names, especially for infrequently used options, and more especially if there's an abbreviation for them. So things like `--verify-signatures` and `--verification-warnings-fatal` are fine with me.